docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+In this getting started guide, you will use Machine ID to create a bot user for
+a machine and use that identity to connect to said machine.
+
+Here's an overview of what you will do:
+
+1. Download and install Teleport (=teleport.version=)
+2. Create bot user
```

Comment:
```suggestion
2. Create a bot user
```

Reply:
The overall tone is conversational (as it should be), so I think it's okay to put an article here.